packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -629,7 +616,7 @@ const ScrollAreaViewportImpl = React.forwardRef(function ScrollAreaViewportImpl(
 
   return (
     <div
-      {...(selector ? getSelectorObj(selector + '-position') : undefined)}
+      data-radix-scroll-area-viewport-position=""
```

Comment:
inlined for now, but will prob eventually be fixed by exposing one or more elements